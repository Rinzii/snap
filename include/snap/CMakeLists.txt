snap_add_headers(
        inplace_vector.hpp
        numbers.hpp
        span.hpp
)

add_subdirectory(bit)
add_subdirectory(concepts)
add_subdirectory(debugging)
add_subdirectory(expected)
add_subdirectory(functional)
add_subdirectory(internal)
add_subdirectory(memory)
add_subdirectory(meta)
add_subdirectory(numeric)
add_subdirectory(type_traits)
add_subdirectory(utility)
