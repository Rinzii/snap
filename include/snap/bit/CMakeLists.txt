snap_add_headers(
        bit_cast.hpp
        bit_ceil.hpp
        bit_floor.hpp
        bit_width.hpp
        byteswap.hpp
        countl.hpp
        countr.hpp
        endian.hpp
        has_single_bit.hpp
        popcount.hpp
        rotl.hpp
        rotr.hpp
)
