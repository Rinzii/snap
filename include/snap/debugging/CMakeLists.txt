snap_add_headers(
        breakpoint.hpp
        breakpoint_if_debugging.hpp
        is_debugger_present.hpp
)

