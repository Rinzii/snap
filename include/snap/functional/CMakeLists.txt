snap_add_headers(
        bind_back.hpp
        bind_front.hpp
        identity.hpp
        invoke_r.hpp
        is_reference_wrapper.hpp
)
