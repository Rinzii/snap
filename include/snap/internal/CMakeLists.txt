

add_subdirectory(builtin)
add_subdirectory(compat)
add_subdirectory(helpers)
add_subdirectory(pp)
