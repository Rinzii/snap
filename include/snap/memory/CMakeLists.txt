snap_add_headers(
        inout_ptr.hpp
        out_ptr.hpp
        retain_ptr.hpp
)
