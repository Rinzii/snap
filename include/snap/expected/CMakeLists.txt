
snap_add_headers(
        unexpected.hpp
)
